### PR TITLE
Enable run button in the source sending panel

### PR DESCRIPTION
**Pull request by Yuyupo**
IoT.jsCode-DCO-1.0-Signed-off-by: Daniella Barsony bella@inf.u-szeged.hu

### CODE REVIEW

**Comment by Yuyupo** [comment]
When you had multiple files open, added some to the Selected file(s) section and closed the ones you didn't add from Available files(s) the Run button was disabled.

File: src/app/session.js [hunk 1]
```diff
@@ -932,7 +932,7 @@ export default class Session {
 
     // Refresh the run panel list.
     if (this._surface.getPanelProperty('run.active')) {
-      this._surface.updateRunPanel(SURFACE_RUN_UPDATE_TYPE.ALL, null, this);
+      this._surface.updateRunPanel(SURFACE_RUN_UPDATE_TYPE.ALL, this.debuggerObj, this);
```
Comment (knightburton) [inline src/app/session.js]: The `debuggerObj` is not available in the current `this`. I think that parameter should be a simple `undefined` or a boolean and in that case the condition in the surface module would be much simple.

**Comment by Yuyupo** [comment]
Changed

**Comment by yichoi** [comment]
LGTM

**Pull request merged**